docs/tutorial.md

```diff
@@ -0,0 +1,290 @@
+API Version:  validation.spectrocloud.labs/v1alpha1
+Kind:         ValidationResult
+Metadata:
+  Creation Timestamp:  2023-11-15T17:22:33Z
+  Generation:          1
+  Resource Version:    141250
+  UID:                 18d37dd9-9e4a-4397-b857-03116cf62975
+Spec:
+  Expected Results:  1
+  Plugin:            AWS
+Status:
+  Conditions:
+    Failures:
+      v1alpha1.IamRoleRule SpectroCloudRole missing action(s): [s3:PutObject s3:DeleteObject s3:PutBucketOwnershipControls s3:PutBucketPolicy s3:PutBucketPublicAccessBlock s3:PutObjectAcl] for resource arn:*:s3:::* from policy Controllers Policy
+    Last Validation Time:  2023-11-15T17:26:37Z
+    Message:               One or more required IAM permissions was not found, or a condition was not met
+    Status:                False
+    Validation Rule:       validation-SpectroCloudRole
+    Validation Type:       aws-iam-role-policy
+  Sink State:              N/A
+  State:                   Failed
+Events:                    <none>
+```
+
+If the validation fails, you can review the `Failures` section of the `Conditions` section to determine the cause of the failure. In this example, the IAM role is missing the required permissions. You can update the IAM role to add the missing permissions and rerun the validation.
+
+
+The Validator will continouslly re-issue a validation cand update the `ValidationResult` CR with the result of the validation. You can use the following command to get the status of the validation. Once you addressed the cause of the failure, the validation will pass.
+
+
+If you encountered an error and fixed the error, after about 30 seconds check the validation results again. In this example, the IAM policy was updated to add the missing permissions. 
+
+
+```shell
+kubectl describe validationresults
+```
+
+```shell
+Name:         validator-plugin-aws-awsvalidator-spectro-cloud-base
+Namespace:    default
+Labels:       <none>
+Annotations:  validator/validation-result-hash: 2A5jQj0W4SBN8IKGC0zRbQ==
+API Version:  validation.spectrocloud.labs/v1alpha1
+Kind:         ValidationResult
+Metadata:
+  Creation Timestamp:  2023-11-15T17:43:11Z
+  Generation:          1
+  Resource Version:    1937
+  UID:                 876b29bd-8d96-47f4-8d0c-c4d6eeae039c
+Spec:
+  Expected Results:  1
+  Plugin:            AWS
+Status:
+  Conditions:
+    Last Validation Time:  2023-11-15T17:53:21Z
+    Message:               All required aws-iam-role-policy permissions were found
+    Status:                True
+    Validation Rule:       validation-SpectroCloudRole
+    Validation Type:       aws-iam-role-policy
+  Sink State:              N/A
+  State:                   Succeeded
+Events:                    <none>
+```
+
+
+The validation results are hashed and result events are only updated if the result has changed. In this example, the validation check was successful and the `ValidationResult` CR was updated with the result of the validation.
```

Comment:
"result events are only updated" -> "result events are only emitted"

A brief intro to sinks would be helpful before this sentence. The ValidationResult CRs are always updated with whatever the latest details are. But the validation controller will only re-emit events to your configured sink if the hash changes.